src/dom.js

```diff
@@ -467,3 +467,13 @@ export function openWindowDialog(win, url, target, opt_features) {
   }
   return res;
 }
+
+/**
+ * Whether the element is a script tag with application/json type.
+ * @param {!Element} element
+ * @return {boolean}
+ */
+export function isJsonScriptTag(element) {
+  return element.tagName == 'SCRIPT' &&
+            element.getAttribute('type').toUpperCase() == 'APPLICATION/JSON';
```

Comment:
I think @jridgewell was referring to `element.tagName` always being reported in uppercase by all browsers. MIME types are not case-sensitive, lower-case is only by convention so I think we should keep the case-insensitive comparison.